lib/Doctrine/Common/Persistence/Mapping/Driver/PHPDriver.php

```diff
@@ -35,7 +35,7 @@
 class PHPDriver extends FileDriver
 {
     /**
-     * {@inheritDoc}
+     * @var ClassMetadata
      */
     protected $metadata;
```

Comment:
Note: there is no `$metadata` in `FileDriver`, so `@inheritDoc` is invalid here.